ibm_cloud_sdk_core/base_service.py

```diff
@@ -81,6 +82,7 @@ def __init__(self,
         self.authenticator = authenticator
         self.disable_ssl_verification = disable_ssl_verification
         self.default_headers = None
+        self.gzip_compression = False
```

Comment:
Why not allow this new flag to be passed in via an optional parameter, similar to disable_ssl_verification?

Reply:
Also need to add "gzip_compression" to the list of attributes in the comments above.

Reply:
Good point, done